AuthenticatorShared/UI/Platform/Settings/Settings/SettingsProcessorTests.swift

```diff
@@ -30,6 +34,24 @@ class SettingsProcessorTests: AuthenticatorTestCase {
 
     // MARK: Tests
 
+    /// Performing `.loadData` with the password manager sync disabled should set
```

Comment:
🎨 Personally, I like to avoid "should" language in test descriptions, and just state declaratively what it does:
```suggestion
    /// Performing `.loadData` with the password manager sync disabled sets
```